db/migration/1680178010470-EnableFacetingForAreaCharts.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class EnableFacetingForAreaCharts1680178010470
+    implements MigrationInterface
+{
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+            UPDATE charts
+            SET config = JSON_REMOVE(config, "$.addCountryMode")
+            WHERE
+                config ->> "$.type" = "StackedArea"
+                AND JSON_LENGTH(config ->> "$.data.availableEntities") > 1
+                AND config ->> "$.addCountryMode" = "change-country"
+                AND (
+                    config ->> "$.selectedEntityIds" IS NULL
+                    OR config ->> "$.selectedEntityIds[0]" = 355 -- 355=World
```

Comment:
Oh no, this is where legacy stuff comes in again 🙊 
`selectedEntityIds` is a legacy property, and is superseded by `selectedEntityNames` where present (which should be most charts I think).

Anyhow, I don't think we need this extra condition at all?